Tests/private_build/tests/test_configure_and_test_integration_instances_private.py

```diff
@@ -36,7 +36,7 @@ class MockApiClient:
     def __init__(self):
         self.configuration = MockConfiguration()
 
-    def call_api(self, resource_path, method, header_params, files, auth_settings):
+    def call_api(self, resource_path, method, header_params, files, auth_settings, response_type):
```

Comment:
No default value for response_type?

Reply:
This is the mock, this value is ignored anyway